nondex-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'groovy'
+apply plugin: 'maven'
+
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+
+    }
+    dependencies {
+        classpath "com.gradle.publish:plugin-publish-plugin:0.9.7"
+    }
+}
+apply plugin: "com.gradle.plugin-publish"
+repositories {
+    jcenter()
+    mavenLocal()
+}
+
+dependencies {
+    compile gradleApi()
+    compile localGroovy()
+    compile 'edu.illinois:nondex-common:1.1.2-SNAPSHOT'
```

Comment:
can you use something like edu.illinois:nondex-common:$version
where $version is the one below that you define.